build-support/check_github_labels.py

```diff
@@ -37,6 +39,10 @@ def check_labels_impolite(pr_num):
 
 
 if __name__ == '__main__':
+    if not is_correct_build_folder():
+        print('This build originated from a jenkins subfolder. Skipping ci-label check')
```

Comment:
Shouldn't this be `LOG.info` ?

Reply:
This whole script uses `print()`. I can switch it to `LOG` if you want.